code/upaya/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rails:4.2.6
+
+MAINTAINER Eric Maland <eric.maland@gsa.gov>
+
+ADD . /upaya
+WORKDIR /upaya
+RUN bundler install --without deploy
```

Comment:
Copy the Gemfile first.. then run bundler, then copy the rest of the app.  That means the bundler command will only be run when the Gemfile changes:

There is the one I'm working on:

FROM rails:4.2.6                                                                 
                                                                                 
MAINTAINER Eric Maland <eric.maland@gsa.gov>                                     
                                                                                 
WORKDIR /upaya                                                                   
RUN mkdir /myapp                                                                 
                                                                                 
COPY Gemfile /upaya                                                              
COPY Gemfile.lock /upaya                                                         
RUN bundler install --without deployment deploy test                             
                                                                                 
COPY . /upaya                                                                    
                                                                                 
EXPOSE 3000                                                                      
CMD ["rails", "server", "-b", "0.0.0.0"]